src/cmds/pbs_ralter.c

```diff
@@ -294,7 +294,7 @@ main(int argc, char *argv[], char *envp[])		/* pbs_ralter */
 			pbs_server, pbs_errno);
 		CS_close_app();
 		exit(pbs_errno);
-	} else
+	} else if (pbs_errno)
```

Comment:
just to confirm, will any pbs_errno be set when there's only 1 instance down and rest are up?

Reply:
no, if any of them are down. If all of them are down then it will hit the if case.